src/BlazorUI/Demo/Client/Core/Pages/Components/ProgressIndicator/BitProgressIndicatorDemo.razor

```diff
@@ -61,4 +61,4 @@ Bars moving backwards reduce confidence in the service."
             await Task.Delay(100);
         }
     }
-}
```

Comment:
please roll back this change, do not remove this empty new line at the end of the file.